packages/x-data-grid/src/models/props/DataGridProps.ts

```diff
@@ -128,30 +128,30 @@ export interface DataGridPropsWithDefaultValues {
    */
   checkboxSelectionVisibleOnly: boolean;
   /**
-   * Number of extra columns to be rendered before/after the visible slice.
-   * @default 3
+   * Column region to render before/after the viewport
+   * @default 100
    */
-  columnBuffer: number;
+  columnBufferPx: number;
   /**
-   * Number of extra rows to be rendered before/after the visible slice.
-   * @default 3
+   * Row region to render before/after the viewport
+   * @default 100
    */
-  rowBuffer: number;
+  rowBufferPx: number;
```

Comment:
I don't love the new name with a `Px` suffix, but leaving the name as it didn't feel like a good option because it would silently make the grid less performant for the users using this prop.

Reply:
`rowBufferRegion` could be another option, JSDoc can explain the unit of the region (px)